server/src/main/java/io/mantisrx/master/jobcluster/JobClusterHelper.java

```diff
@@ -0,0 +1,86 @@
+package io.mantisrx.master.jobcluster;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import io.mantisrx.common.Label;
+import io.mantisrx.runtime.command.InvalidJobException;
+import io.mantisrx.server.master.domain.JobDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class JobClusterHelper {
+
+    public enum SystemLabels {
+        MANTIS_IS_RESUBMIT_LABEL("_mantis.isResubmit"),
+        MANTIS_ARTIFACT_LABEL("_mantis.artifact"),
+        MANTIS_VERSION_LABEL("_mantis.version");
```

Comment:
I have definitely seen `_mantis.artifact.version` in some places, is the UI using the right names? Also, we should capture all system labels here like `_mantis.user`, `_mantis.ownerEmail` etc.